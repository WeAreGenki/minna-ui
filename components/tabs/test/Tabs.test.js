'use strict';

const Tabs = require('../src/Tabs.svelte');

describe('Tabs component', () => {
  it('renders correctly with defaults', () => {
    const target = document.createElement('div');
    new Tabs({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
