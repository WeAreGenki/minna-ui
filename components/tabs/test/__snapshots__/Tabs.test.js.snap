// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs component renders correctly with defaults 1`] = `
"<div class=\\"tabs\\"><nav><a href=\\"#1\\" class=\\"tabs-title tabs-title-active\\">1</a>
    <a href=\\"#2\\" class=\\"tabs-title\\">2</a>
    <a href=\\"#3\\" class=\\"tabs-title\\">3</a>
    <a href=\\"#4\\" class=\\"tabs-title\\">4</a>
    <a href=\\"#5\\" class=\\"tabs-title\\">5</a>
    <a href=\\"#6\\" class=\\"tabs-title\\">6</a></nav>
  <div class=\\"tabs-content\\">  
      </div></div>"
`;
